lib/hanami/middleware/body_parser.rb

```diff
@@ -70,7 +70,7 @@ def build_parsers(parser_names)
       # @api private
       def _symbolize(body)
         if body.is_a?(::Hash)
-          Utils::Hash.deep_symbolize(body)
+          Router::Params.deep_symbolize(body)
         else
```

Comment:
Why a router is responsible for symbolizing keys in a JSON body?

Reply:
It's a separated, optional middleware to mount. The namespace is `Hanami::Middleware` 😉